python/2020-10-02-default_bids_stage_Config.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'default_bids_stage', 'Default stage to use in session when importing EEG/IEEG data from BIDS dataset', 1, 0, 'text', ID, 'Default stage for BIDS dataset', 25 FROM ConfigSettings WHERE Name="imaging_pipeline";
```

Comment:
maybe this should be "default import stage" ? @cmadjar is this possibly relevant as an argument for the imaging uploader/pipeline too?